src/html/escaped_string.cr

```diff
@@ -0,0 +1,34 @@
+class HTML::EscapedString
+  SUBSTITUTIONS = {
+    '&'  => "&amp;"
+    '\"' => "&quot;",
+    '\'' => "&apos;",
+    '<'  => "&lt;",
+    '>'  => "&gt;",
+  }
+
+  def self.escape(str)
+    new(str).to_s
+  end
+
+  def initialize(@str)
+    @escaped_str = StringIO.new
```

Comment:
Any reason for not making this a local variable of the `escaped` method? It looks like it does not need to be an instance variable.